### ISSUE TITLE
ValueError: invalid literal for int() with base 10: 'False'

### ISSUE THREAD

**Issue by KDragonM** [Mikadily/poloniexlendingbot#241]
I tested the code on the my raspberry pi 3  and ran into this error and confirmed that the error still exists on windows 10 also. I am using the up to date code that I freshly cloned today with the default config apart from the API key and secret.

Here's the error I encountered both times:
```
pi@raspberrypi:~/Programs/poloniexlendingbot $ python lendingbot.py
Traceback (most recent call last):
  File "lendingbot.py", line 55, in <module>
    Lending.init(Config, api, log, Data, MaxToLend, dry_run, Analysis, notify_conf)
  File "/home/pi/Programs/poloniexlendingbot/modules/Lending.py", line 78, in init
    scheduler.enter(10, 1, notify_summary, (int(notify_conf['notify_summary_minutes']) * 60, ))
ValueError: invalid literal for int() with base 10: 'False'
```

**Comment by laxdog** [comment]
I missed that in testing... 

If you set a number in your config file for notify_summary_minutes it should fix that. 

I'll make a PR to fix it properly next week, I'm on holiday at the minute.

**Comment by KDragonM** [comment]
Thanks, that fixed that error but there are some error messages after that point. I'll make a new issue with the error.

**Comment by rnevet** [comment]
Making a fix for this, there is no reason to run the scheduler if Notifications aren't configured.

**Comment by Evanito** [comment]
Proposed fixed by #243
